tes/models.py

```diff
@@ -294,7 +290,7 @@ def is_valid(self) -> Tuple[bool, Union[None, TypeError]]:
             for e in self.executors:
                 if e.image is None:
                     errs.append("Executor image must be provided")
-                if len(e.command) == 0:
+                if e.command is None or len(e.command) == 0:
```

Comment:
I think this additional check is necessary, or otherwise an error with traceback is thrown when the user/client passes `command=None` - which is rather inconsistent, as we otherwise nicely collect all the (common) problems that may arise.